### DISCUSSION TITLE
nuxt 3 keep alive pages

### DISCUSSION THREAD

**Discussion by nserekolouis** [nuxt/nuxt#27431] [Questions]
I have am loading a list to which users can click items and navigate to a page showing more content on that list. I want to cache the page so that I do not have to reload the page all the time.

**Comment by xjccc** [comment]
look at this?

[keepalive](https://nuxt.com/docs/api/nuxt-config#keepalive)

[definePageMeta-keepalive](https://nuxt.com/docs/api/utils/define-page-meta#parameters)

**Comment by nserekolouis** [comment]
I decided to use stores to retrieve data.

Nsereko Louis
Full Stack Software Developer
MTN: +256773294701.
Airtel: +256706096087.
